lib/trycourier/session.rb

```diff
@@ -0,0 +1,85 @@
+require "net/http"
+require "json"
+require "openssl"
+
+module Courier
+  class CourierAPISession
+
+    def initialize(base_url)
+      @base_url = base_url
+      @auth_token = nil
+      @username = nil
+      @password = nil
+      @auth_method = nil
```

Comment:
An instance variable in ruby will by default evaluate to nil, so these are unnecessary.

```suggestion
```